CHANGELOG.md

```diff
@@ -26,6 +26,8 @@ If possible, provide tooling that performs the changes, e.g. a shell-script.
 
 * When using subcommand parsers, e.g. `git push`, typing `git puhs -h` will raise an exception that the user misspelled
   the subcommand instead of printing the help page of `git` ([\#172](https://github.com/seqan/sharg-parser/pull/172)).
+* Using the `sharg::output_file_validator` to validate a directory deletes the directory's contents
```

Comment:
This sounds like "now it deletes the contens". But we now avoid that content is deleted right?

```suggestion
* The `sharg::output_file_validator` will throw if the given path is a directory and writeability is not checked as it would delete the directory's contents.
```